README.md

```diff
@@ -20,11 +20,42 @@ Using the stock OSX version of git will likely cause problems. You probably want
     homesick symlink  dotfiles
 
 ### Optional
-If you're going to use hitch install it by running bundle in the dotfiles directory
+
+If you're going to use hitch or other ruby scripts in our dotfiles, install gem
+dependencies by running bundle in the dotfiles directory
```

Comment:
keep in mind that you'll only have the bundled gems in the gemset and ruby version specified by the `.ruby-{gemset,version}` files. You may need to do this command in other ruby versions depending on how you do your project workflow.